lib/nodejs/serializer.js

```diff
@@ -131,7 +134,12 @@ class SerializableEvent {
    */
   constructor(eventName, originalValue, originalError) {
     if (!eventName) {
-      throw new Error('expected a non-empty `eventName` string argument');
+      throw createInvalidArgumentValueError(
```

Comment:
if `eventName` is `undefined` (e.g., someone wrote `new SerializableEvent()`) then one could argue this is  an invalid _type_ (it should be a `string`; not `undefined`) instead of an invalid _value_.  contrast that with passing `''` for `eventName`--the type is correct but the value is invalid.  in this case, we're checking for _falsy_, which is both a type and a value (because JavaScript)--and is intended--so I don't feel strongly that we need two different errors here.  just wanted to mention it

further, these two types of errors are unlikely to be caused by an end-user--it's an error only Mocha contributors should ever encounter.  we don't usually do this sort of exhaustive type-checking unless it's user input

Reply:
Makes sense to me. I wasn't tied to the two different error types, so I've gone ahead and changed this one to throw a `type` error too.